packages/react-integration/demo-app-ts/src/components/demos/TopologyDemo/components/DemoTaskNode.tsx

```diff
@@ -68,6 +70,7 @@ const DemoTaskNode: React.FunctionComponent<DemoTaskNodeProps> = ({
         {...passedData}
         {...rest}
         badgePopoverParams={badgePopoverParams}
```

Comment:
I think it's worth communicating somehow, that when data.badgeTooltips is true, then the badgePopoverParams are not going to be used anymore. Right now, they are constant and appear to always be used, but that is not the case.